spring-boot/autoconfigure/src/main/java/com/linecorp/armeria/spring/ThriftServiceRegistrationBean.java

```diff
@@ -17,12 +17,16 @@
 
 import java.util.ArrayList;
 import java.util.Collection;
+import java.util.function.Function;
 
+import javax.annotation.Nonnull;
 import javax.validation.constraints.NotNull;
```

Comment:
Java has way too many nullability annotations that mean the same thing... This one under validation triggers spring's checker and is appropriate for beans I think.